packages/opentelemetry-core/package.json

```diff
@@ -10,7 +10,7 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts --exclude 'test/platform/browser/**/*.ts'",
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts' --exclude 'test/platform/browser/**/*.ts'",
```

Comment:
is this needed for some reason? Not blocking just curious

Reply:
It was needed at one point in the PR. I had a test in the test directory that wouldn't run without them, subdirectories were fine. I can remove the quotes as I moved the test. FWIW, some packages have quotes, and others do not.